New log file opened at Fri May 08 11:20:49 CEST 2015 by 80052@Laban
Taking next connection to: 55521
New client attempting to connect on port: 55522-55521
  Address = /130.242.47.35:55521
This client on port 55522 has uuid 9ac97b13-e967-492c-bf3d-7ce1733678b2 configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-9ac97b13-e967-492c-bf3d-7ce1733678b2
Taking next connection to: 55521
New client attempting to connect on port: 55523-55521
  Address = /130.242.47.35:55521
Handshake exception on socket: 55523
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 55521
New client attempting to connect on port: 55524-55521
  Address = /130.242.47.35:55521
This client on port 55524 has uuid c7ed59ae-0079-4171-81c1-87ac36ca2a17 configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-9ac97b13-e967-492c-bf3d-7ce1733678b2, activator-c7ed59ae-0079-4171-81c1-87ac36ca2a17
Taking next connection to: 55521
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 55521
New client attempting to connect on port: 55564-55521
  Address = /130.242.47.35:55521
Handshake exception on socket: 55564
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 55521
New client attempting to connect on port: 55565-55521
  Address = /130.242.47.35:55521
This client on port 55565 has uuid 92fd2bd8-911e-4576-9d24-fbfd81383740 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-c7ed59ae-0079-4171-81c1-87ac36ca2a17, play-fork-run-92fd2bd8-911e-4576-9d24-fbfd81383740
Taking next connection to: 55521
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 55521
